### ISSUE TITLE
internal link needed

### ISSUE THREAD

**Issue by viperbbb** [elias-sundqvist/obsidian-annotator#326]
future request: internal link can link to the obsidian note, there is no way to link to other note now.

**Comment by maxwibert** [comment]
I second this request. It would be great to be able to link to other notes in Obsidian.

**Comment by Deus333** [comment]
Yes, there's no way to make links from pages that this plugin shows it's doable in regular pdf files but not here. I think it's a missing and crucial feature for those that want to work with pdfs in Obsidian.

**Comment by vfairon** [comment]
Yep ! Came here with the intention of asking for it ! Would be extremely helpful !

**Comment by harrydabble** [comment]
Hey guys, any news on this? It would make this plugin complete.
